### PR TITLE
Revise Train Scripts [Latte and DiT]: avoid setting positional embedding to trainable parameters

### PR DESCRIPTION
**Pull request by wtomin**
For DiT and Latte models, the positional embedding tensor is initialized as a Parameter that does not require gradient updates.

However, in the training script, I accidently set all parameters to trainable. It is not desirable for positional embeddings, because they are initialized and then fixed during training.

### CODE REVIEW

**Comment by zhtmike** [comment]
maybe just change to tensor and skip load the pos embed parameter, prevent accident change all parameter to be trainable

**Comment by wtomin** [comment]
> maybe just change to tensor and skip load the pos embed parameter, prevent accident change all parameter to be trainable

Now the pos_embed and temp_embed (in latte only) are Tensors instead of Parameters.

File: mindone/models/latte.py [hunk 1]
```diff
@@ -81,8 +84,8 @@ def __init__(
             )
         num_patches = self.x_embedder.num_patches
         # Will use fixed sin-cos embedding:
-        self.pos_embed = ms.Parameter(ops.zeros((1, num_patches, hidden_size), dtype=ms.float32), requires_grad=False)
-        self.temp_embed = ms.Parameter(ops.zeros((1, num_frames, hidden_size), dtype=ms.float32), requires_grad=False)
+        self.pos_embed = ops.zeros((1, num_patches, hidden_size), dtype=ms.float32)
```
Comment (zhtmike) [inline mindone/models/latte.py:87]: just change to `self.pos_embed = Tensor(pos_embed).float().unsqueeze(0)` here for easier reading

**Review by zhtmike — APPROVED** [review]

**Review by hadipash — APPROVED** [review]

**Review by SamitHuang — APPROVED** [review]

**Review by SamitHuang — APPROVED** [review]
